### PR TITLE
On 431 create component

### PR DESCRIPTION
**Pull request by shahin-tl**
https://tradeledger.atlassian.net/browse/ON-431

### CODE REVIEW

File: tl-app/src/main/java/io/tradeledger/registrylookup/config/bean/AbstractAwsDevelopmentContext.java [hunk 1]
```diff
@@ -23,12 +23,10 @@ protected final RemoteParamsService doCreateRemoteParamsService() {
         final Map<String, String> values = Maps.newHashMap();
 
         // api credentials
-        values.put("/approvals_api/platform_pass", "Cere600!");
+        values.put("/registry_lookup_api/platform_pass", "Cere600!");
```
Comment (winnie-kang) [inline tl-app/src/main/java/io/tradeledger/registrylookup/config/bean/AbstractAwsDevelopmentContext.java:26]: DId we setup a component user in shield api?

Comment (shahin-tl) [reply tl-app/src/main/java/io/tradeledger/registrylookup/config/bean/AbstractAwsDevelopmentContext.java:26]: yup

File: tl-app/src/main/java/io/tradeledger/registrylookup/model/companiesHouse.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package io.tradeledger.registrylookup.model;
+
+public class companiesHouse {
```
Comment (winnie-kang) [inline tl-app/src/main/java/io/tradeledger/registrylookup/model/companiesHouse.java:3]: Capital C here

File: tl-app/src/main/resources/config/application.yml [hunk 3]
```diff
@@ -4,11 +4,11 @@ spring.profiles.active: development
...
 server.shutdown: graceful
 spring.lifecycle.timeout-per-shutdown-phase: ${SHUTDOWN_TIMEOUT_PERIOD:30s}
-ratpack.port: 5210
+ratpack.port: 5200
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 tradeledger:
     services:
-        componentUriPrefix: '/companysearch'
+        componentUriPrefix: '/registrylookup'
         platformUser: "test+syscompappr@tradeledger.io"
```
Comment (winnie-kang) [inline tl-app/src/main/resources/config/application.yml:12]: this is the approval platform user. I believe we need to change to the one for regsitry lookup.

File: tl-app/src/main/java/io/tradeledger/registrylookup/model/companiesHouse.java (hunk 2 above)
Comment (shahin-tl) [reply tl-app/src/main/java/io/tradeledger/registrylookup/model/companiesHouse.java]: thanx. fixed.

File: tl-app/src/main/resources/config/application.yml (hunk 3 above)
Comment (shahin-tl) [reply tl-app/src/main/resources/config/application.yml]: fixed .. thanks.

**Review by winnie-kang — APPROVED** [review]

**Pull request merged**